### ISSUE TITLE
How to Avoid SteamVR to Get Vive Tracker Tracking Data？

### ISSUE THREAD

**Issue by Dongyongupupup** [ValveSoftware/openvr#1628]
How to Avoid SteamVR to Get Vive Tracker Tracking Data？

**Comment by Rectus** [comment]
Do you want to completely bypass the SteamVR application? If so, you would need a custom driver for the hardware. 

Check out https://github.com/cntools/libsurvive/

**Comment by Dongyongupupup** [comment]
> Do you want to completely bypass the SteamVR application? If so, you would need a custom driver for the hardware.
> 
> Check out https://github.com/cntools/libsurvive/

Thanks for your suggestion！It helped me a lot. If there is something I don't understand, I will ask you again.

**Comment by YongD0ng** [comment]
Can I bypass SteamVR completely with an ios device to get Vive tracker tracking data？

**Comment by okawo80085** [comment]
> Can I bypass SteamVR completely with an ios device to get Vive tracker tracking data？

Considering that SteamVR dropped support for IOS, I'd say libsurvive is your best bet, it's originally made for Linux iirc, so it should require minimal setup in IOS

**Comment by YongD0ng** [comment]
As far as I know, the ios serial port api is closed to libsurvive

**Comment by YongD0ng** [comment]
> As far as I know, the ios serial port api is closed to libsurvive

**Comment by okawo80085** [comment]
Well i did say minimal setup, not no setup, if it doesn't have permissions to access that API and it *needs* it to work, you'll either have to make it work or you're out of luck.

**Comment by felixshing** [comment]
Hello guys, I would like to ask how can I get the position and orientation from Vive Tracker? Is it the same way as getting the data from the headset and controllers?

**Comment by Rectus** [comment]
Yes, with OpenVR it should work the same as with other tracked devices.
